getting_started/setup.adoc

```diff
@@ -0,0 +1,148 @@
+
+The following sections describe the system requirements per component type.
+
+=== Master
+
+Masters must meet the following system requirements:
+
+- Physical or virtual system
+ifdef::openshift-origin[]
+- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+endif::[]
+ifdef::openshift-enterprise[]
+- Base OS: Red Hat Enterprise Linux Server 7.0
+endif::[]
+- 2 vCPU
+- Minimum 8 GB RAM
+- Minimum 30 GB hard disk space
+
+=== Nodes
+
+Nodes must meet the following system requirements:
+
+- Physical or virtual system, or an instance running on a public IaaS
+ifdef::openshift-origin[]
+- Base OS: Fedora 21, CentOS 7, or Red Hat Enterprise Linux Server 7.0
+endif::[]
+ifdef::openshift-enterprise[]
+- Base OS: Red Hat Enterprise Linux Server 7.0
+endif::[]
+- 1 vCPU
+- Minimum 8 GB RAM
+- Minimum 15 GB hard disk space
+
+
+=== Other Beta Requirements
+
+- You must have a GitHub account or a local (on premise) Git server.
+- A shared network must exist between the master and node components.
+
+== Prerequisites
+
+Whether installing OpenShift using RPM packages or Docker images, you must first satisfy the following prerequisites.
+
+ifdef::openshift-enterprise[]
+=== Installing Red Hat Enterprise Linux 7
+
+As mentioned in the system requirements, a base installation of Red Hat Enterprise Linux (RHEL) 7.0 is required for master or node hosts. More information on installing RHEL 7.0 can be found in the following documentation:
+
+https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/Installation_Guide/index.html[Red Hat Enterprise Linux 7 Installation Guide]
+
+=== Registering with Red Hat Network
+
+Each system must be registered to Red Hat Network (RHN) and have a RHEL subscription attached to access required packages.
+
+. Register the system and attach a RHEL subscription:
++
+----
+# subscription-manager register --username=<rhnuser> --password=<rhnpassword>
+# subscription-manager list --available   (Find pool ID for RHEL subscription)
+# subscription-manager attach --pool=<pool_id>
+----
+
+. Ensure the base RHEL repository is enabled, as well:
++
+----
+# subscription-manager repos --enable=rhel-7-server-rpms
+----
+
+. At this point, you can update the system to the latest RHEL base packages:
++
+----
+# yum update
+----
+endif::[]
+
+=== Security Warning
+
+OpenShift is a system which runs Docker containers on your system. In some cases (for example, for build operations and the registry service), it does so using privileged containers. Furthermore, those containers access your host's Docker daemon and perform `docker build` and `docker push` operations. As such, you should be aware of the inherent security risks associated with performing `docker run` operations on arbitrary images as they effectively have root access. This is particularly relevant when running the OpenShift nodes directly on your host system.
+
```

Comment:
[Minor suggestion for readability]
OpenShift runs Docker containers on your system, and in some cases, such as build operations and the registry service, it does so using privileged containers.